demo-shell/src/app.config.json

```diff
@@ -969,6 +969,35 @@
       "delete"
     ]
   },
+  "adf-edit-service-task-filter": {
```

Comment:
maybe we can make these as defaults and avoid adding more content to app config? but document how to customize?

Reply:
This is the extension of it, if missing the component cascades to its default values specified  inside the component itself https://github.com/Alfresco/alfresco-ng2-components/blob/9da3ec90d0fa830167e6cbb361e65e7b1dd68b41/lib/process-services-cloud/src/lib/task/task-filters/components/edit-task-filters/edit-task-filter-cloud.component.ts#L43-L44